src/highlight/__tests__/HighlightAnnotations-test.tsx

```diff
@@ -140,7 +152,7 @@ describe('HighlightAnnotations', () => {
     describe('handleAnnotationActive()', () => {
         test('should call setActiveAnnotationId', () => {
             const wrapper = getWrapper();
-            wrapper.find(HighlightList).simulate('select', '123');
+            (wrapper.find(HighlightList).prop('onSelect') as Function)('123');
```

Comment:
Is the cast needed here and for `onClick`? It looks like it's not needed in other tests.

Reply:
It is because the `onSelect` prop is typed as optional

Reply:
We should probably allow the non-null assertion in tests. It's actually safer than a cast, in this case, since it maintains the true type. Can be done later, though.